.github/workflows/angular-builders.yml

```diff
@@ -18,12 +18,12 @@ jobs:
         with:
           token: ${{ secrets.GH_PERSONAL_TOKEN }}
 
-      - uses: actions/cache@v1
+      - uses: actions/cache@v2
```

Comment:
I think it's better to keep it as a separate PR, in order to avoid mixing features.